darasimiolaifa/traveller-fin-compass
# feat(Landing page): implement conversion app landing page view

## What does this PR do?
This PR implements the UI/UX component of the app

## What major activities were carried out?
- create app-info, conversion display, conversion form, and conversion manipulator view components
- create styling using CSS rules

## How can this be manually tested?
- Clone the repository
- Fetch the branch
- Install the dependencies
- cd into the `traveller-fin-compass` folder
- Start project by running `npm start` in the terminal
- Your browser should automatically load to show you the homepage screen below:

![Landing Page](https://user-images.githubusercontent.com/28527393/72203534-106ed500-346d-11ea-8a11-0b2ec4945bb4.png)

PT Story Link: [#168503682](https://www.pivotaltracker.com/story/show/168503682)
